tests/test_crosstalk.py

```diff
@@ -282,7 +285,8 @@ def testTaskAPI_NL(self):
         correction.
         """
         for this_isr_task in [IsrTask, IsrTaskLSST]:
-            self.checkTaskAPI_NL(this_isr_task)
+            for subtrahendMasking in [False, True]:
+                self.checkTaskAPI_NL(this_isr_task, subtrahendMasking)
```

Comment:
Does `checkSubtracted` do enough here? In addition to adding another call to this test, would it be possible to add something that checks the mask planes are being cleared properly along the way (i.e., the original bug that prompted all this)?

Reply:
Added a region that is declared "saturated" with matching test to confirm that masks aren't being duplicated.  I can confirm that running the test from this ticket with `ip_isr` on main throws a `AssertionError: 100 != 25` (25 pixels for this saturated region, x4 due to that mask being duplicated across all four amplifiers).